src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_StringConcat.cs

```diff
@@ -397,6 +706,21 @@ private BoundExpression ConvertConcatExprToString(SyntaxNode syntax, BoundExpres
                 }
             }
 
+            // If expression is of form `constantChar.ToString()` then rewrite it here to a string literal so we can choose better options for lowering later (e.g. fold it with another constant instead of performing concatenation)
```

Comment:
>(e.g. fold it with another constant instead of performing concatenation) [](http://example.com/codeflow?start=158&length=72)

Consider splitting this line, it is significantly longer than the other lines in this comment